proposals/lip-0024.md

```diff
@@ -207,4 +207,4 @@ If the implementation of this proposal is done at height `h` then we additionall
 
 ## Reference Implementation
 
-TBD
+[Add fail safe delegate banning mechanism](https://github.com/LiskHQ/lisk-sdk/issues/5058)
```

Comment:
I did not have a closer look, but it seems like the implementation you link is mainly about productivity banning, which was already part of LIP 0023, and not about the PoM transaction defined in this LIP.